ranger/gui/ui.py

```diff
@@ -339,7 +344,7 @@ def redraw(self):
             self.console.focused = False
             self.console.visible = False
             self.status.visible = True
-
+        
```

Comment:
Remove this trailing whitespace.

Reply:
Same like the white space before.

Reply:
Not sure what you mean? Blank lines should be completely blank. The diff shows spaces.

Reply:
I meant it happened by accident and that I'll remove it :) like the one in line 405 ;)